Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/RemoteMongoClientIntTests.swift

```diff
@@ -20,15 +20,9 @@ class RemoteMongoClientIntTests: BaseStitchIntTestCocoaTouch {
     private var mongoClient: RemoteMongoClient!
     
     override func setUp() {
-//        guard mongodbUri != nil && mongodbUri != "<your-mongodb-uri>" else {
```

Comment:
Don't you want this code but uncommented?

Reply:
nah now he does `private lazy var mongodbUri: String = pList?[mongodbUriProp] as? String ?? "mongodb://localhost:26000"` earlier in the file